### DISCUSSION TITLE
Isolating selectors on className

### DISCUSSION THREAD

**Discussion by tcolinpa** [tailwindlabs/tailwindcss#10922] [Help] [answered]
Is there a way to isolate selectors instead of just repeating it to apply classes?

How it's: `before:absolute before:inset-0`
To something likeish: `before:(absolute inset-0)`

Is there another way to make it less verbose in complex components?

**Comment by wongjn** [comment, marked as answer]
Nope!

[Here's a response from a maintainer for a similar question](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314

[Also another thread here](https://github.com/tailwindlabs/tailwindcss/discussions/8337#discussioncomment-4036320) with more responses from maintainers.

**Reply by Matin7315m** [reply]
And also for sizes?
(sm , lg , xl &...) like :
sm:bg-black sm:text-white sm:rounded
